backend/development.ini.sample

```diff
@@ -388,10 +388,10 @@ email.reply.activated = False
 ### IMAP ###
 # Tracim email reply feature will read all mails in following IMAP,
 # to be able to transform email responses into Tracim app comments.
-email.reply.imap.server = your_imap_server
-email.reply.imap.port = 993
-email.reply.imap.user = your_imap_user
-email.reply.imap.password = your_imap_password
+; email.reply.imap.server = your_imap_server
+; email.reply.imap.port = 993
+; email.reply.imap.user = your_imap_user
+; email.reply.imap.password = your_imap_password
 email.reply.imap.folder = INBOX
 email.reply.imap.use_ssl = True
```

Comment:
This two line don't have default parameter ?

Reply:
fixed.